classes/class3/hdavidzhu_class3_homework/db.js

```diff
@@ -0,0 +1,6 @@
+var mongoose   = require('mongoose');
+
+var mongoURI = process.env.MONGOURI || "mongodb://olinjs_user:olinjs_password@ds053648.mongolab.com:53648/olinjs";

```

Comment:
You should not hardcode your mongolab URI. This is now publicly available for all time (in your commit history). This one may not be super secure, but other databases may be.
Additionally, this is the purpose of having the environment variable, the mongolab URI will be set by that on Heroku and you will use the string locally.